packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -140,6 +140,7 @@ function getLightSlottedContent(rootNodes: IrChildNode[], cxt: TransformerContex
                     break;
                 }
                 // SlottableLeafIrType
+                case 'Slot':
```

Comment:
If you add this here, then you also need to update `SlottableLeafIrType` above.

I was not able to figure out how to do the TypeScript jiu-jitsu to get a single source of truth for this. 😛